.github/workflows/publish_v3.yml

```diff
@@ -9,6 +9,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      VERSION: v3.0
```

Comment:
maybe no discussion for this PR: we will have patch releases? Shouldn't that be v3.0.0? Especially in the persistent URL for the context.

Reply:
`v3.0` is still hard coded in multiple places, e.g. line 18